src/style/style.css

```diff
@@ -0,0 +1,20 @@
+@import "./card.css";
+@import "./star.css";
+@import "./price.css";
+@import "./variable.css";
+@import "./button.css";
+@import "./product.css";
+
+html {
+  font-family: "Roboto", sans-serif;
+  line-height: 18px;
+  font-style: normal;
+  font-size: 16px;
+  font-weight: 400;
+}
+
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
use `*` selector only for box sizing property
```suggestion
```

Reply:
i did